applications/openshift/kubelet/kubelet_configure_tls_cipher_suites_openshiftapiserver_operator/rule.yml

```diff
@@ -0,0 +1,42 @@
+documentation_complete: true
+
+prodtype: ocp4
+
+title: "Ensure that the OpenShift API Server only makes use of Strong Cryptographic Ciphers"
+
+description: |-
+  Ensure that the OpenShift API Server is configured to only use strong cryptographic ciphers.
```

Comment:
Same comment here, this is covering the OpenShift API Server Operator TLS config.